FreeAPS/Sources/Localizations/Main/en.lproj/Localizable.strings

```diff
@@ -1950,7 +1950,7 @@ Enact a temp Basal or a temp target */
 "Max COB" = "Max COB";
 
 /* "Max COB" */
-"The default of maxCOB is 120. (If someone enters more carbs in one or multiple entries, Trio will cap COB to maxCOB and keep it at maxCOB until the carbs entered above maxCOB have shown to be absorbed. Essentially, this just limits UAM as a safety cap against weird COB calculations due to fluky data.)" = "The default of maxCOB is 120. (If someone enters more carbs in one or multiple entries, Trio will cap COB to maxCOB and keep it at maxCOB until the carbs entered above maxCOB have shown to be absorbed. Essentially, this just limits UAM as a safety cap against weird COB calculations due to fluky data.)";
+"The maximum amount of carbs that can be bloused or set high temp targets for. This safety feature protects against erroneous carbohydrate entries that could lead to hypoglycemia episodes." = "The maximum amount of carbs that can be bloused or set high temp targets for. This safety feature protects against erroneous carbohydrate entries that could lead to hypoglycemia episodes.";
```

Comment:
Typo: “bloused”

Reply:
This setting does not set the maximum amount of carbs that can be entered, it just caps the COB to limit oref. Max Carbs is in ⚙️ > Meal Settings

So if your Max COB is 120g, and you enter 150g of carbs, your COB will only increase to 120g but it will remain at 120g until oref determines you've absorbed 31g (either by your glucose trends or by timing out).

Reply:
Ah interesting, I didn't love the initial way this was phrased, had trouble and looked at our docs, and saw

"The maximum amount of carbs that Trio is allowed to bolus or set high temp targets for. This safety feature protects against erroneous carbohydrate entries that could lead to hypoglycemia episodes.".

We should update this too.

Reply:
"The default of maxCOB is 120. If more carbs are entered, COB will be capped to maxCOB until the excess carbs have been absorbed by glucose trends or timing out."